### DISCUSSION TITLE
A beta testing mode

### DISCUSSION THREAD

**Discussion by MEGA-Dawg68** [SethBurkart123/EvenBetterSEQTA#45] [Ideas]
exactly what is sounds like, if there are big features coming (like the AI for example) we could definitely use a beta mode for those individuals who are willing to have a potentially broken SEQTA.
This is how I think it should work

1. Make sure people know the license, "THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY"

2. It isn't a new extension on chrome webstore or at least public, so no general user stumbles upon it and breaks something

3. Gotta contribute somehow, if that is feedback or fixes for broken code

4. Help anyone who has problems with normal EvenBetterSEQTA because if you are using a beta then you probably know a thing or two

What do you think :)

**Comment by SethBurkart123** [comment]
that actually sounds interesting. Something I could do is post another version to the web store as the 'beta' branch with all of the experimental features, and then just have a link to it via github. That way it would auto-update.

**Reply by MEGA-Dawg68** [reply]
Are you able to hide it from public view? I have only posted to the web store never fiddled with settings

**Reply by SethBurkart123** [reply]
yes, there is an option to make it unlisted

**Reply by MEGA-Dawg68** [reply]
Then that would be awesome! Im willing to sacrifice SEQTA for this lol

**Reply by ar-cyber** [reply]
> Then that would be awesome! Im willing to sacrifice SEQTA for this lol

same!

**Reply by SethBurkart123** [reply]
I might have a bit of a look into this when I'm working on the next i guess "big thing"... I've now got it set up so I can basically instantly push to the chrome web store from the command line which means its super easy to push a little like (beta) update. So it's actually probs quite doable.... :)

**Comment by ar-cyber** [comment]
I mean `npm run dev` does exist :) (and building from source or unfinished branches)

**Reply by SethBurkart123** [reply]
keep in mind that isn't automatic updates.. that's kinda the whole idea around this.
